generator/template.go

```diff
@@ -195,6 +196,105 @@ function getNotifyEntityArrivalSink<T>(notifyCallback: NotifyStreamEntityArrival
     }
   })
 }
+
+type Primitive = string | boolean | number;
+type RequestPayload = Record<string, unknown>;
+type FlattenedRequestPayload = Record<string, Primitive | Array<Primitive>>;
+
+function isPlainObject(val: unknown): boolean {
```

Comment:
can you add a comment here stating the source of the logic?

Reply:
Understood. Will do so 🙇🏼